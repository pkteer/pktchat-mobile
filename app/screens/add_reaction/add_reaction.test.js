// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';

import {shallowWithIntl} from '@test/intl-test-helper';

import AddReaction from './index';

describe('AddReaction', () => {
    const baseProps = {
        onEmojiPress: jest.fn(),
        componentId: 'component-id',
        closeButton: {},
    };

    test('should match snapshot', () => {
        const wrapper = shallowWithIntl(<AddReaction {...baseProps}/>);

        expect(wrapper.getElement()).toMatchSnapshot();
    });
});
