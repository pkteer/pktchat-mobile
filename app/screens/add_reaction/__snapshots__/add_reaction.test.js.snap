// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddReaction should match snapshot 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
  testID="add_reaction.screen"
>
  <Connect(EmojiPicker)
    deviceWidth={750}
    onEmojiPress={[Function]}
    testID="add_reaction.emoji_picker"
  />
</View>
`;
